Themis/ViewModels/Assessment/Modeling/UMLRendererViewModel.swift

```diff
@@ -9,12 +9,15 @@ import Foundation
 import SharedModels
 import Common
 import SwiftUI
+import ApollonModels
+import ApollonView
 
 class UMLRendererViewModel: ExerciseRendererViewModel {
     @Published var umlModel: UMLModel?
     @Published var selectedElement: SelectableUMLItem?
     @Published var error: Error?
     @Published var currentDragLocation = CGPoint.zero
+    @Published var offset: CGPoint = CGPoint(x: 15, y: 15)
```

Comment:
To make SwiftLint happy 😅 
```suggestion
    @Published var offset = CGPoint(x: 15, y: 15)
```